amida-tech/recruitment-registry
# Feature/rr 590

If this PR fixes a bug, you _must_ add test cases representative of the bug.

Please refer to [Tettra](https://app.tettra.co/teams/amida/pages/amida-pull-request-and-code-review-guide) for PR review guidelines.

#### What does this PR do? Add ability to use year range in criteria for search

#### Related JIRA tickets: RR-590

#### How should this be manually tested? 

1) Use seed profile survey which has year of birth.
2) Create a filter with yearRange.
3) Test with various cohorts to make sure list of patients are filtered by year range.

#### Background/Context NA

#### Screenshots (if appropriate): NA
